src/main/java/com/minecolonies/core/colony/buildings/moduleviews/RequestTaskModuleView.java

```diff
@@ -38,4 +41,10 @@ public String getDesc()
     {
         return "com.minecolonies.coremod.gui.workerhuts.crafter.tasks";
     }
+
+    /**
+     * Get the specific task list.
+     * @return the task list.
+     */
+    public abstract List<IToken<?>> getTasks();
```

Comment:
if this is all you need, I think you should be using an interface and not creating a class hierarchy

Reply:
The other stuff in the abstract is also needed